src/views/accounts/account-details/account-details.tsx

```diff
@@ -57,10 +57,7 @@ export function AccountDetails() {
           )}
 
           {activeTabName === TabNames.multisigSettings && (
-            <MultisigSettings
-              accountInfo={data?.accountInfo}
-              accountAddress={accountAddress}
-            />
+            <MultisigSettings accountAddress={accountAddress} />
```

Comment:
IIRC you need to be `owner`, not just `canMultisigSubmit`, to be able to change settings per transaction. Can you not show the form if the user isn't owner (or disable it)?

Reply:
this jsx shows the settings under the "Multisig Settings" tab that anyone can view.  If they try to modify it without an owner role, they'll see this error below.  Although, we could just remove the "save" button as well.
![image](https://user-images.githubusercontent.com/7282254/177377976-778013cf-b266-452f-ab2e-13f847cf1b3c.png)

In this pr currently, when trying to create a transaction, the form will be shown but it will be disabled if the user does not have the "owner" role.  those indicators are here:
https://github.com/tantommy/albert/blob/a67f5373fe3ef4e48300427eb8b32a159cb70c0f/src/features/transactions/components/send-asset/send-asset.tsx#L133-L136